Dockerfile

```diff
@@ -14,15 +14,10 @@
 
 FROM --platform=$BUILDPLATFORM golang:1.16.8-stretch as builder
 WORKDIR /go/src/github.com/kubernetes-sigs/aws-fsx-csi-driver
-
-ARG TARGETOS
-ARG TARGETARCH
-RUN echo "TARGETOS:$TARGETOS, TARGETARCH:$TARGETARCH"
-RUN echo "I am running on $(uname -s)/$(uname -m)"
-
 COPY . .
-
-RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} make aws-fsx-csi-driver
+ARG OS
```

Comment:
I'd rather use explicit OS and ARG than magic docker variables TARGETOS and TARGETARCH.